app/views/insured/families/_enrollment_actions.html.erb

```diff
@@ -2,6 +2,7 @@
 
 <% if @bs4 %>
 <div class="enrollment-actions-ddl">
+  <% key_index = (0...8).map { (65 + rand(26)).chr }.join %>
```

Comment:
Hm... I see what this does... I wonder if it makes more sense to just use the `enrollment.id` to enforce the uniqueness of the links instead?

Reply:
Good call. will switch tonight